examples/J21_to_rates_test.txt

```diff
@@ -0,0 +1,26 @@
+0.000000e+00 4.331219e-12 7.106207e-12 8.926086e-13 2.999175e-11 1.052004e-10 1.298340e-11
+2.000000e-01 4.049527e-12 5.827013e-12 6.472565e-13 2.582831e-11 8.068766e-11 9.166991e-12
+4.000000e-01 3.800945e-12 4.797898e-12 4.698398e-13 2.241988e-11 6.218828e-11 6.478486e-12
+6.000000e-01 3.580203e-12 3.965890e-12 3.414067e-13 1.960654e-11 4.816210e-11 4.582796e-12
+8.000000e-01 3.383043e-12 3.290073e-12 2.483324e-13 1.726576e-11 3.747761e-11 3.244885e-12
+1.000000e+00 3.205994e-12 2.738676e-12 1.808108e-13 1.530306e-11 2.930038e-11 2.299754e-12
+1.200000e+00 3.046211e-12 2.286894e-12 1.317755e-13 1.364510e-11 2.301265e-11 1.631464e-12
+1.400000e+00 2.901343e-12 1.915257e-12 9.612905e-14 1.223458e-11 1.815525e-11 1.158479e-12
+1.600000e+00 2.769436e-12 1.608403e-12 7.018976e-14 1.102643e-11 1.438546e-11 8.234041e-13
+1.800000e+00 2.648855e-12 1.354150e-12 5.129579e-14 9.984964e-12 1.144644e-11 5.858038e-13
+2.000000e+00 2.538221e-12 1.142786e-12 3.752049e-14 9.081719e-12 9.144872e-12 4.171619e-13
+2.200000e+00 2.436368e-12 9.665368e-13 2.746778e-14 8.293863e-12 7.334615e-12 2.973509e-13
+2.400000e+00 2.342303e-12 8.191445e-13 2.012503e-14 7.602949e-12 5.904732e-12 2.121507e-13
+2.600000e+00 2.255173e-12 6.955532e-13 1.475699e-14 6.993977e-12 4.770633e-12 1.515055e-13
+2.800000e+00 2.174246e-12 5.916593e-13 1.082920e-14 6.454676e-12 3.867537e-12 1.082975e-13
+3.000000e+00 2.098887e-12 5.041187e-13 7.952859e-15 5.974943e-12 3.145617e-12 7.748403e-14
+3.200000e+00 2.028544e-12 4.301959e-13 5.844777e-15 5.546412e-12 2.566384e-12 5.548895e-14
+3.400000e+00 1.962736e-12 3.676447e-13 4.298541e-15 5.162112e-12 2.099982e-12 3.977389e-14
+3.600000e+00 1.901040e-12 3.146147e-13 3.163537e-15 4.816208e-12 1.723149e-12 2.853534e-14
+3.800000e+00 1.843086e-12 2.695761e-13 2.329777e-15 4.503782e-12 1.417690e-12 2.049074e-14
+4.000000e+00 1.788543e-12 2.312606e-13 1.716865e-15 4.220676e-12 1.169316e-12 1.472711e-14
+4.200000e+00 1.737121e-12 1.986135e-13 1.265987e-15 3.963350e-12 9.667587e-13 1.059395e-14
+4.400000e+00 1.688561e-12 1.707552e-13 9.340829e-16 3.728780e-12 8.010980e-13 7.627360e-15
+4.600000e+00 1.642631e-12 1.469504e-13 6.895978e-16 3.514371e-12 6.652483e-13 5.496187e-15
+4.800000e+00 1.599124e-12 1.265830e-13 5.093924e-16 3.317884e-12 5.535596e-13 3.963824e-15
+5.000000e+00 1.557853e-12 1.091351e-13 3.764847e-16 3.137382e-12 4.615109e-13 2.861064e-15
```

Comment:
How is this file generated? What if we want to change the UVB?

Reply:
This is the conversion between spectral slopes (left column) and ionisation/heating rates at J21=1. It was calculated by performing the integral over the spectrum (and cross section) above each ionisation threshold. I could include this code as a util file somewhere.

Reply:
Thanks I'd really appreciate that.